lib/matplotlib/patches.py

```diff
@@ -733,6 +739,9 @@ def __init__(self, xy, width, height, angle=0.0, **kwargs):
         self._width = width
         self._height = height
         self.angle = float(angle)
+        self.rotate_around_center = rotate_around_center
+        # Required for RectangleSelector with axes aspect ratio != 1
```

Comment:
Could you expand on this comment a bit to describe what the value of `_aspect_ratio_correction` means?